templates/company/officers/list.html.tx

```diff
@@ -270,35 +270,45 @@
     % }
 
     % if $officers.total_results > $paging.entries_per_page {
-        <ul class="pager">
+        <nav class="govuk-!-margin-top-6 govuk-!-margin-bottom-0 govuk-pagination" aria-label="Pagination">
             % if ($paging.previous_page) {
-                <li>
-                    <a class="page" href="<% $c.url_for %>?page=<% $paging.previous_page %>">
-                        Previous
+                <div class="govuk-pagination__prev">
+                    <a class="govuk-link govuk-pagination__link" href="<% $c.url_with.query([ page => $pager.previous_page ]) %>" rel="prev">
+                    <svg class="govuk-pagination__icon govuk-pagination__icon--prev" xmlns="http://www.w3.org/2000/svg" height="13" width="15" aria-hidden="true" focusable="false" viewBox="0 0 15 13">
+                        <path d="m6.5938-0.0078125-6.7266 6.7266 6.7441 6.4062 1.377-1.449-4.1856-3.9768h12.896v-2h-12.984l4.2931-4.293-1.414-1.414z"></path>
+                    </svg>
+                        <span class="govuk-pagination__link-title">
+                            Previous<span class="govuk-visually-hidden"> page</span>
+                        </span>
                     </a>
-                </li>
+                </div>
             % }
+            <ul class="govuk-pagination__list">
             % for $paging.page_set -> $page {
-                <li>
-                    <a class="page" href="<% $c.url_for %>?page=<% $page %>">
+            <li class="govuk-pagination__item govuk-!-display-block <% if $paging.current_page_number { %> govuk-pagination__item--current <% } %> ">
```

Comment:
`<% if $paging.current_page_number { %>`

Should this be

`<% if $paging.current_page_number == $page { %>`

This may be an issue elsewhere.